debugProtocol.json

```diff
@@ -1273,7 +1273,7 @@
 					"description": "The value of the variable."
```

Comment:
After your comment, I realized that using setVariable is not ideal, because a real debugger would need a frameId to know what context to resolve the expression against. I updated the proposal to use "evaluate" to change the value.